### PR TITLE
Dao idls pda getters

### PR DESCRIPTION
**Pull request by Cain310**
- Add DAO idls and pda getters

## Checklist

- [ ] UAT
- [ ] Passes final QA checks

### CODE REVIEW

File: src/dao/constants.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+};
+
+/**
+ * Star Atlas DAO Account addresses.
+ */
+export const DAO_ADDRESSES = {
+  REGISTERED_LOCKER: new PublicKey(
+    'EAgUhz4LG8fqtrShFo1NDrvEiNPxEi7kHe96D76NzkAj'
+  ),
+  LOCKER: new PublicKey('5WmM9c9WE71y78Ah8Bp8vgyoStscM1ZZyaaFqRf8b2Qa'),
```
Comment (remjx) [inline src/dao/constants.ts:19]: Can we create a namespace or some naming scheme to distinguish between ATLAS and POLIS locker variables? One step in this direction would be to nest everything inside an `atlas-locker` folder, but we should make sure there is no confusing overlap when people are importing functions/modules. Perhaps in the `index.ts` export all of these under a `polisLocker` namespace.

File: src/dao/pda_getters/proxy_rewarder_Pda.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { STAR_ATLAS_ADDRESSES } from '../constants';
+
+/**
+ * Returns the public key for the proxy account
+ */
+export const findProxyAddress = async (
+  escrowKey: PublicKey,
+  user: PublicKey
+): Promise<[PublicKey, number]> => {
+  return await PublicKey.findProgramAddress(
```
Comment (remjx) [inline src/dao/pda_getters/proxy_rewarder_Pda.ts:12]: `await` is redundant if you are returning a promise. (There are other cases of this in the code, but just making a single comment for brevity. You can find them by searching `return await`)
```suggestion
  return PublicKey.findProgramAddress(
```

File: src/dao/constants.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { PublicKey } from '@solana/web3.js';
+
+/**
+ * Star Atlas DAO Program addresses.
+ */
+export const STAR_ATLAS_ADDRESSES = {
```
Comment (remjx) [inline src/dao/constants.ts:6]: I don't see a need to separate `STAR_ATLAS_ADDRESSES` and `DAO_ADDRESSES`, can we combine them into a single var?

File: src/dao/pda_getters/locked_voter_Pda.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { PublicKey } from '@solana/web3.js';
+import * as Seeds from './seeds';
+import { STAR_ATLAS_ADDRESSES, DAO_ADDRESSES } from '../constants';
+
+/**
+ * Finds the address of an Escrow.
```
Comment (remjx) [inline src/dao/pda_getters/locked_voter_Pda.ts:6]: Is there a follow-up task to add more than just these basic `getters` to factory?

File: src/dao/pda_getters/seeds.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export const PROXY_SEED = Buffer.from('Proxy');
```
Comment (remjx) [inline src/dao/pda_getters/seeds.ts:1]: Move these to `constants.ts`?

File: src/dao/types/lockedVoterIdl.ts [hunk 6]
```diff
@@ -0,0 +1,346 @@
+export type LockedVoterIDL = {
+  version: '0.1.0';
```
Comment (remjx) [inline src/dao/types/lockedVoterIdl.ts:2]: This version does not align with the version in the `polis-dao` repo
```
export type LockedVoterIDL = {
  version: "0.5.6";
  ```
Did you manually set this version number? I think we should keep them in sync.

**Review by remjx — CHANGES_REQUESTED** [review]

File: src/dao/pda_getters/seeds.ts (hunk 5 above)
Comment (Cain310) [reply src/dao/pda_getters/seeds.ts:1]: ok

File: src/dao/pda_getters/locked_voter_Pda.ts (hunk 4 above)
Comment (Cain310) [reply src/dao/pda_getters/locked_voter_Pda.ts:6]: I think the follow up to this will be when/if we decide to add the wrappers and/or function calls.

**Pull request closed without merging**